src/functionBased/pages/About.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable */
+
+import React from 'react';
+import { Link, useRouteMatch, Route } from 'react-router-dom';
+import SinglePage from './SinglePage';
+
+const About = (props) => {
+  console.log(useRouteMatch);
```

Comment:
remember to remove console.log from your finished project

Reply:
Thank you for the review @Kalunge I have implemented some of the changes.